crypto/finite_field.ts

```diff
@@ -126,6 +126,9 @@ function createField(p: bigint, t: bigint, twoadicRoot: bigint) {
     xor(x: bigint, y: bigint) {
       return x ^ y;
     },
+    and(x: bigint, y: bigint) {
+      return x & y;
+    },
```

Comment:
It looks like [we'll pull this out into its own library](https://github.com/o1-labs/o1js-bindings/pull/180#discussion_r1362285209) sometime in the future, but I assume this is alright for now?

Reply:
Oh! That makes a lot of sense 😂 ! Should I eliminate this and do it all in TS with the infix operator (I could also update Florian's implementation), or should we just leave it? (I don't mind changing!)

Reply:
I vote yes!